garak/cli.py

```diff
@@ -489,6 +489,11 @@ def main(arguments=None) -> None:
                 f"generators.{_config.plugins.model_type}", config_root=_config
             )
 
+            if generator.parallel_capable and not _config.system.parallel_attempts:
```

Comment:
Might want to suppress this if a config file was provided, while the message could be helpful for a cli hint if `-- config` was provided on the command line the hint is likely unnecessary.

It looks like `args` has already been clobbered by the time we get to this code so a mechanism to detect if `--config` was passed would need to be added earlier.

Reply:
agree - private var added to track this & condition hint display